v2/cli/term/event_page.go

```diff
@@ -113,7 +108,9 @@ func (e *eventPage) refresh(eventID string) {
 			switch evt.Rune() {
 			case 'r', 'R': // Reload
 				e.router.loadEventPage(eventID)
-			case 'w', 'W':
+			case 'l', 'L':
+				e.eventLogs.SetText("Placeholder logs")
+				go e.streamEventLog(eventID)
```

Comment:
This is problematic because you're launching a goroutine and giving yourself no way to end that goroutine. I'd really hold off on trying to stream at this point. If you get the UI elements working, we can pair on how to do the streaming cleanly.

Reply:
Ok sounds good, I'll remove these calls for now